src/pages/Lend/Filter/FilterSectionAttributes.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<filter-menu-section title="Attributes" class="filter-section-attributes">
+		<ais-refinement-list
+			:attribute="'themeData.loanThemeTypeName'"
+			:sort-by="['name:asc']"
+			:transform-items="transformItems"
+		/>
+	</filter-menu-section>
+</template>
+
+<script>
+import FilterMenuSection from '@/pages/Lend/Filter/FilterMenuSection';
+import { AisRefinementList } from 'vue-instantsearch';
+
+export default {
+	components: {
+		FilterMenuSection,
+		AisRefinementList,
+	},
+	data() {
+		return {
+			visibleThemes: {
+				'conflict zone': true,
+				'clean energy': true,
+				'displaced populations': true,
+				'rural exclusion': true,
+				'social enterprise': true,
```

Comment:
This doesn't exist in our current indices. "Small Enterprise" does though...

Reply:
Let's merge this then we can update this list as necessary